source/css/scss/_palette.scss

```diff
@@ -141,4 +141,7 @@ $letter-colors: (
 
 .kiva-darkgreen-bg {
 	background-color: $kiva-darkgreen;
-}
\ No newline at end of file
+}
+
+// panel stuff
+$callout-panel-bg: scale-color($vivid-yellow, $lightness: 50%);
```

Comment:
fixed; the vivid yellow is nicer anyways.  removed the lightening of the background and the darkening of the border. @jedlin-kiva